core/program_info.h

```diff
@@ -48,6 +48,15 @@ class ProgramInfo : public ICastable {
 
     /// @returns the FuzzerConfig associated with this program.
     [[nodiscard]] const FuzzerConfig &getFuzzerConfig() const;
+
+    /// Proceeds to override the fuzzer configurations if a TOML file is provided (checked in either
+    /// `Bmv2V1ModelRtSmithTarget` or `TofinoTnaRtSmithTarget`).
+    void proceedToOverrideFuzzerConfigsViaFile(const char *path);
```

Comment:
You can skip the proceed. Just call this function `loadFuzzerConfig`. Path should be a std::filesystem::path object. Not a char string. This applies to all other uses.